README.md

```diff
@@ -162,6 +162,13 @@ server-tools: # Here we repeat server-tools, but no problem because it's a
     - html_text
 ```
 
+You can add all modules in a repo by using a `*`:
+
+```yaml
+website:
+    - "*"
```

Comment:
Works for me - I'll go with `website: "*"` so that the code works without changes, due to strings being treated as arrays.